src/legacy/ui/public/visualize/loader/pipeline_helpers/utilities.ts

```diff
@@ -30,21 +30,21 @@ import { fieldFormats } from '../../../registry/field_formats';
 
 const config = chrome.getUiSettingsClient();
 
-const defaultFormat = { convert: identity };
```

Comment:
Weird thing I noticed when writing tests: the way we had defined the default formatter, the same object was being passed to every field that used the default format, which meant mutating `convert` on one field would actually change all the default fields.

I wasn't sure how likely it would be for this to cause problems in the real world, but went ahead and changed it anyway just to be safe.